docs/procedures.md

```diff
@@ -46,28 +45,21 @@ Execute the following commands on your local machine, not production.
 
 2.  Make sure you have configured repositories:
 
-    - `origin` for your repository on GitHub
-    - `upstream` for [`carpentries/amy` repo on GitHub](https://github.com/carpentries/amy)
+    - `origin` for [`carpentries/amy` repo on GitHub](https://github.com/carpentries/amy)
 
-    For example, this is the correct configuration for `chrismedrela`:
+    For example, this is the correct configuration:
 
         $ git remote -v
-        origin	git@github.com:chrismedrela/amy.git (fetch)
-        origin	git@github.com:chrismedrela/amy.git (push)
-        upstream	git@github.com:carpentries/amy.git (fetch)
-        upstream	git@github.com:carpentries/amy.git (push)
+        origin	git@github.com:carpentries/amy.git (fetch)
+        origin	git@github.com:carpentries/amy.git (push)
 
 3.  Make sure your local `develop` and `main` branches are up to date:
```

Comment:
Good spot - we should have some guidance here about what release branch naming is/should look like. From our discussions, it looks like `develop` will also end up being used in our DEV testing workflow, rather than UAT, so it makes sense to update the docs to reflect this now?

Reply:
I've updated this to use a release branch instead